docs/index.asciidoc

```diff
@@ -160,57 +177,58 @@ input {
 -----
 
 [id="plugins-{type}s-{plugin}-interval"]
-===== `interval` 
+===== `interval`
+
+  * Value type is <<number,number>>
+  * Default value is `30`
 
 The `interval` option specifies the polling interval in seconds.
 If polling all configured hosts takes longer than this interval, a warning will be emitted to the logs.
 
-* Value type is <<number,number>>
-* Default value is `30`
-
 [id="plugins-{type}s-{plugin}-mib_paths"]
-===== `mib_paths` 
+===== `mib_paths`
 
-The `mib_paths` option specifies the location of one or more imported MIB files. The value can be either a dir path containing
-the imported MIB `.dic` files or a file path to a single MIB `.dic` file.
+  * Value type is <<path,path>>
+  * There is no default value for this setting
 
-* Value type is <<path,path>>
-* There is no default value for this setting
+The `mib_paths` option specifies the location of one or more imported MIB files.
+The value can be either a dir path containing the imported MIB `.dic` files or a
+file path to a single MIB `.dic` file.
```

Comment:
Nice reorg to standardize the format with other options... here and other places as needed.  Thanks!